bottles/frontend/views/bottle_preferences.py

```diff
@@ -139,6 +137,36 @@ def __init__(self, details, config, **kwargs):
         self.queue = details.queue
         self.details = details
 
+        if not gamemode_available or not Xdp.Portal.running_under_sandbox():
+            return
+
+        if not gamescope_available:
+            _gamescope_not_available = (
+                f"{_flatpak_not_available} {_flatpak_pkg_name['gamescope']}"
+            )
+            self.switch_gamescope.set_tooltip_text(_gamescope_not_available)
+            self.btn_manage_gamescope.set_tooltip_text(_gamescope_not_available)
+
+        if not vkbasalt_available:
+            _vkbasalt_not_available = (
+                f"{_flatpak_not_available} {_flatpak_pkg_name['vkbasalt']}"
+            )
+            self.switch_vkbasalt.set_tooltip_text(_vkbasalt_not_available)
+            self.btn_manage_vkbasalt.set_tooltip_text(_vkbasalt_not_available)
+
+        if not mangohud_available:
+            _mangohud_not_available = (
+                f"{_flatpak_not_available} {_flatpak_pkg_name['mangohud']}"
+            )
+            self.switch_mangohud.set_tooltip_text(_mangohud_not_available)
+            self.btn_manage_mangohud.set_tooltip_text(_mangohud_not_available)
+
+        if not obs_vkc_available:
+            _obsvkc_not_available = (
+                f"{_flatpak_not_available} {_flatpak_pkg_name['obsvkc']}"
+            )
+            self.switch_obsvkc.set_tooltip_text(_obsvkc_not_available)
+
```

Comment:
```
Traceback (most recent call last):
  File "/app/share/bottles/bottles/frontend/windows/main_window.py", line 223, in set_manager
    self.page_details = DetailsView(self)
                        ^^^^^^^^^^^^^^^^^
  File "/app/share/bottles/bottles/frontend/views/details.py", line 81, in __init__
    self.view_preferences = PreferencesView(self, config)
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/share/bottles/bottles/frontend/views/bottle_preferences.py", line 145, in __init__
    f"{_flatpak_not_available} {_flatpak_pkg_name['gamescope']}"
       ^^^^^^^^^^^^^^^^^^^^^^
UnboundLocalError: cannot access local variable '_flatpak_not_available' where it is not associated with a value
```

Reply:
How are you running into this bug?

Reply:
Ah okay, looks like it happens when an extension isn't installed.